src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -129,6 +129,7 @@ class MediaBaseViewer extends BaseViewer {
         return this.getRepStatus()
             .getPromise()
             .then(() => {
+                this.startLoadTimer();
```

Comment:
Could we move the startLoadTimer to a part of  the promise chain in RepStatus.js? We could `then` it over there once and avoid doing it everywhere else. Basically if the repstatus is success then we want to start timing the content load.

Reply:
This is not meant to track time RepStatus. This begins the timer for when we setup and send the GET request for the representation. Inside of RepStatus, you'll see a timer related to representation conversion time

Reply:
Right, but since we want to track from once rep status comes back to success until the file is done loading, why not append to the end of rep status so we only have to add it in one place?

Reply:
Oh, I see, I misunderstood your Q. We don't want to do that because we don't want to include the time to process the JS between the rep coming back and then making the request for the rep. We ONLY want to be tracking network related times, for that metric.